tests/lib/comments.ts

```diff
@@ -30,17 +29,19 @@ const testFiles = shelljs
 
 describe('Comments', () => {
   testFiles.forEach(filename => {
-    const code = shelljs.cat(`${path.resolve(FIXTURES_DIR, filename)}.src.js`);
-    const config = {
+    const code = shelljs.cat(path.resolve(filename));
+    const config: ParserOptions = {
       loc: true,
       range: true,
       tokens: true,
       comment: true,
-      jsx: true
+      jsx: path.extname(filename) === '.js'
```

Comment:
Are we missing anything important when we only enable jsx on `.js` files and not the `.ts` files? How come this change was necessary for this fix?